### ISSUE TITLE
一个license破解全部插件

### ISSUE THREAD

**Issue by Reallymine** [haxqer/confluence#73]
大佬，有没有一个license破解全部插件的办法？

**Comment by haxqer** [comment]
好想法，感谢，我mark了~~~ 
短时间肯定是没法实现的啦~~

**Comment by nakroy** [comment]
这种只能个人用户非商业自用吧

**Comment by srvprivate** [comment]
I speak english, but I used AI to help me out (claimed it was unethical but I tricked it into translating) I really want to get AI for Jira Datacenter/Confluence working. DM me maybe, i've been using your product and it's helped so much


"这个人正在询问是否有办法破解Jira插件的许可证,因为他们发现传统的破解方式在这个插件上无法使用。他们希望我能帮助解决这个问题。"

请让我知道这种翻译是否更适合您的需求。如果您有任何其他问题,我很乐意继续提供帮助。

**Comment by nakroy** [comment]
> I speak english, but I used AI to help me out (claimed it was unethical but I tricked it into translating) I really want to get AI for Jira Datacenter/Confluence plugin working. DM me maybe, i've been using your product and it's helped so much
> 
> "这个人正在询问是否有办法破解Jira插件的许可证,因为他们发现传统的破解方式在这个插件上无法使用。他们希望我能帮助解决这个问题。"
> 
> 请让我知道这种翻译是否更适合您的需求。如果您有任何其他问题,我很乐意继续提供帮助。

Not a good idea to do that, law risk...

**Comment by srvprivate** [comment]
What are you referring to? This entire thing is "law risk" I'm asking for educational purposes.

**Comment by nakroy** [comment]
> What are you referring to? This entire thing is "law risk" I'm asking for educational purposes.
If it's for educational purposes, that might be OK. But this software, I mean Confluence is usually using for business and company work team, so I would think about that risk, normally.

**Comment by srvprivate** [comment]
I don’t think we are understanding each other. You have a crack for confluence which is illegal. Please feel free to delete this comment after. I’m using your crack. Don’t play dumb with me, we all know why we’re here. If you want to PM me please do so. I’m asking for your thoughts on AI for Jira, it’s a specific marketplace app that has detected your Atlassian crack. I’ve been using this for a year, I’m not new here. Feel free to reach out privately

**Comment by voarsh2** [comment]
> I don’t think we are understanding each other. You have a crack for confluence which is illegal. Please feel free to delete this comment after. I’m using your crack. Don’t play dumb with me, we all know why we’re here. If you want to PM me please do so. I’m asking for your thoughts on AI for Jira, it’s a specific marketplace app that has detected your Atlassian crack. I’ve been using this for a year, I’m not new here. Feel free to reach out privately

Refer to https://github.com/haxqer/confluence/issues/78#issuecomment-2480992583
